Bendy and the Dark Revival/bendyandthedarkrevival.asl

```diff
@@ -109,8 +109,22 @@ onStart
 update
 {
     current.IsLoadingSection = vars.Helper.Read<IntPtr>(current.gm + 0xD0) != IntPtr.Zero;
-    current.IsPaused = current.PauseMenuActive && current.GameState == 4 && current.GMIsPaused && current.IsPauseReady;
```

Comment:
this references three pause-specific fields we create in TryLoad, you can remove those definitions to improve startup slightly / since this is unused its not necessary